cli/admin.go

```diff
@@ -70,6 +72,7 @@ type AdminBannerCommand struct {
 	GlobalOpts            *Options `no-flag:"true"`
 	Message               string   `long:"message" short:"m" description:"content of new message"`
 	Clear                 bool     `long:"clear" description:"cleat the banner"`
+	Theme                 string   `long:"theme" short:"t" description:"color theme to use for banner"`
```

Comment:
you may also want to validate this using the choice tag:

choice:         limits the values for an option to a set of values.
                This tag can be specified multiple times (optional)